vms/platformvm/block/builder/helpers_test.go

```diff
@@ -237,7 +237,7 @@ func addSubnet(t *testing.T, env *environment) {
 
 func defaultState(
 	t *testing.T,
-	validators validators.Manager,
+	cfg *config.Config,
```

Comment:
we need config.Config back in state.State so that we can pick the right codec version for metadata depending on whether Durango is activated.